tests/test_config_parsing/test_ensemble_config.py

```diff
@@ -4,62 +4,72 @@
 from ert._c_wrappers.config import ConfigValidationError
 from ert._c_wrappers.enkf import ConfigKeys, ResConfig
 
-from .config_dict_generator import config_dicts, to_config_file
+from .config_dict_generator import config_generators, to_config_file
 
 
-@pytest.mark.usefixtures("use_tmpdir")
-@given(config_dicts())
-def test_ensemble_config_from_file_and_dict_coincide(config_dict):
+@given(config_generators())
+def test_ensemble_config_from_file_and_dict_coincide(
+    tmp_path_factory, config_generator
+):
     filename = "config.ert"
-    to_config_file(filename, config_dict)
+    with config_generator(tmp_path_factory, filename) as config_dict:
+        to_config_file(filename, config_dict)
```

Comment:
Perhaps the contextmanager should perform `to_config_file` for you ?

Reply:
:100:  true